client/my-sites/site-settings/form-general.jsx

```diff
@@ -509,39 +545,40 @@ const FormGeneral = React.createClass( {
 
 		return (
 			<FormFieldset>
-				<legend>{ this.translate( 'Holiday Snow' ) }</legend>
-				<ul>
-					<li>
-						<FormLabel>
-							<FormCheckbox name="holidaysnow" checkedLink={ this.linkState( 'holidaysnow' ) }/>
-							<span>{ this.translate( 'Show falling snow on my blog until January 4th.' ) }</span>
-						</FormLabel>
-					</li>
-				</ul>
+				<legend>{ translate( 'Holiday Snow' ) }</legend>
+				<FormToggle
+					className="is-compact"
+					checked={ !! this.state.holidaysnow }
+					disabled={ this.state.fetchingSettings }
+					onChange={ this.handleToggle( 'holidaysnow' ) }>
+					<span>{ translate( 'Show falling snow on my blog until January 4th.' ) }</span>
+				</FormToggle>
 			</FormFieldset>
 		);
 	},
 
 	Timezone() {
+		const { translate } = this.props;
+
 		if ( this.props.site.jetpack ) {
 			return;
 		}
 
 		return (
 			<FormFieldset>
 				<FormLabel htmlFor="blogtimezone">
-					{ this.translate( 'Site Timezone' ) }
+					{ translate( 'Site Timezone' ) }
 				</FormLabel>
 
 				<Timezone
-					valueLink={ this.linkState( 'timezone_string' ) }
-					selectedZone={ this.linkState( 'timezone_string' ).value }
+					value={ this.state.timezone_string }
```

Comment:
Is `value` used in this component? It looks like only `selectedZone` is used. I think even the `valueLink` that was here was ignored.